src/services/user.js

```diff
@@ -105,7 +115,12 @@ const userService = {
     }
 
     filteredUpdateData.mainSubjects = { ...user.mainSubjects, [role]: updateData.mainSubjects }
-    filteredUpdateData.videoLink = { ...user.videoLink, [role]: updateData.videoLink }
+
+    if (Object.keys(updateData).includes('videoLink'))
```

Comment:
Maybe would be better to use `'videoLink' in updateData` so that js doesn't need to use loops (`Object.keys` and `includes`)?

Reply:
Yeah, it would be much better, but i decided to check directely for the key, because when user deletes videolink it won't be changes, since nothing will be in `updateData.videolink`

Reply:
Nice observation, thank you! Fixed